### ISSUE TITLE
Question about license notice

### ISSUE THREAD

**Issue by swinslow** [stegu/perlin-noise#4]
Hello @stegu, thanks very much for making this available!

I had a question about the license notice from several of the files. I see that it says the following in some files (like sdnoise1234.h):

```
 * This library is public domain software, released by the author
 * into the public domain in February 2011. You may do anything
 * you like with it. You may even remove all attributions,
 * but of course I'd appreciate it if you kept my name somewhere.
 * 
 * This library is distributed in the hope that it will be useful,
 * but WITHOUT ANY WARRANTY; without even the implied warranty of
 * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
 * General Public License for more details.
```

The last line, referring to the GPL -- was that intended to say that the GPL still applies to these files? I assume not, given the public domain language above, but just wanted to make sure about what you intended.

Thanks again!

**Comment by stegu** [comment]
Oops. Sorry for the confusion. I only intended to keep the disclaimer, not
the reference to the GPL text as such.
 I will change that. Thanks for pointing it out!

**Comment by swinslow** [comment]
No worries at all -- thank you @stegu!

**Comment by swinslow** [comment]
Fixed in ff6e625 and subsequent commits -- closing issue. Thank you again @stegu!
